src/Lumi/Components/Select.purs

```diff
@@ -451,6 +452,7 @@ styles = jss
               , lineHeight: "32px"
               , flex: "1 1 0%"
               , display: "block"
+              , position: "absolute"
```

Comment:
In narrow select inputs, the space taken up by the placeholder could cause weird reflows when focusing and leaving the input. This just makes it float under/below the cursor if they're ever visible at the same time.

Reply:
Not sure I follow this. Does the placeholder then float outside the select box?

Reply:
Inside, it just floats over the inner input element. I should test overly-long placeholder messages though.

Reply:
Ah I see. I misunderstood the first comment 😅